.github/workflows/release-app.yml

```diff
@@ -17,23 +20,47 @@ jobs:
       - id: get-matrix
         run: echo "nextcloud_versions=$(./get-matrix.sh)" | tee -a "$GITHUB_OUTPUT"
 
+  build-devcontainer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Pre-build dev container image
+        uses: devcontainers/ci@v0.3
+        with:
+          imageName: ghcr.io/gdatasoftwareag/nextcloud/builder
+          cacheFrom: ghcr.io/gdatasoftwareag/nextcloud/builder
+          push: always
+
   test:
     runs-on: ubuntu-latest
-    needs: define-matrix
+    container:
+      image: ghcr.io/gdatasoftwareag/nextcloud/builder:latest
+      env:
+        IS_CI: true
+      options: --name nextcloud-antivirus-build-container
```

Comment:
i had to set a fixed container name, because I need to put the build container into a network that it can reach the nextcloud and smtp containers.